style.css

```diff
@@ -4,7 +4,6 @@ html {
 
```

Comment:
The popup is resizing and showing properly on mobile and desktop sizes 🤯 The effects are looking nice💣